lib/jobs/linux-catalog.js

```diff
@@ -46,6 +46,7 @@ function catalogJobFactory(
         this.commandUtil = new CommandUtil(this.nodeId);
         this.commands = this.commandUtil.buildCommands(options.commands);
         assert.arrayOfObject(this.commands);
```

Comment:
there is linux-command job schema in lib/task-data/schemas, please add the new option there